sydney/sydney.py

```diff
@@ -7,6 +7,7 @@
 import websockets.client as websockets
 from aiohttp import ClientSession, TCPConnector
 from websockets.client import WebSocketClientProtocol
+import urllib.parse
```

Comment:
Nitpick: Please use isort and then black in order to keep the imports (and the code in general) tidy